docs/docker/redirects.conf

```diff
@@ -0,0 +1,18 @@
+# This is a list of the Nginx rewrite rules that impact only the Documentation
+# pages (pages that URL path start with /docs/...).
+#
+# The blog about the creating the Nginx rewrite rules:
+#  - https://www.nginx.com/blog/creating-nginx-rewrite-rules/
+# The official rewrite module documentation:
+#  - http://nginx.org/en/docs/http/ngx_http_rewrite_module.html
+#
+# Some examples:
+#  - Redirect the page permanently (using HTTP 301 code) from
+#    https://handsontable.com/docs/api/very-old-collection to https://handsontable.com/docs/api/new-collection
+# (uncomment the line bellow to use the rule)
+#rewrite ^/api/very-old-collection/?$ /docs/api/new-collection/ permanent;
+#
+#  - Redirect the page temporary (using HTTP 302 code) from
+#    https://handsontable.com/docs/11.0/api/temp-collection to https://handsontable.com/docs/11.0/api/other-page
+# (uncomment the line bellow to use the rule)
+#rewrite ^/11.0/api/temp-collection/?$ /docs/11.0/api/other-page/ permanent;
```

Comment:
I guess that we already have some manual redirection rules. Can they be moved to here for clarity?